### PR TITLE
GORA-615: Add gora-mongodb settings to gora-tutorial

### PR DESCRIPTION
**Pull request by sneceesay77**
This is now completed and tested.

### CODE REVIEW

File: gora-tutorial/conf/gora.properties [hunk 1]
```diff
@@ -34,18 +34,18 @@ gora.datastore.autocreateschema=true
 #gora.cassandrastore.servers=localhost:9160
 
 ##JDBC properties for gora-sql module using HSQL
-gora.sqlstore.jdbc.driver=org.hsqldb.jdbcDriver
+#gora.sqlstore.jdbc.driver=org.hsqldb.jdbcDriver
```
Comment (djkevincr) [inline gora-tutorial/conf/gora.properties]: Shall we remove these changes? I dont think we need them as part of fix. And this is not related to issue we address with this PR.

File: gora-tutorial/conf/gora-mongodb-mapping.xml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+	<field name="referrer" docfield="referrer" type="string"/>
+	<field name="userAgent" docfield="userAgent" type="string"/>
+    </class>
+
+   <class name="org.apache.gora.tutorial.log.generated.MetricDatum" keyClass="java.lang.String" document="Metrics">
+	    <field name="metricDimension" docfield="metricDimension"  type="string"/>
+	    <field name="timestamp" docfield="timestamp" type="date"/>
+	    <field name="metric" docfield="metric" type="string"/>
+   </class>
+
```
[the remark this answers is not in the document]
Comment (sneceesay77) [reply gora-tutorial/conf/gora-mongodb-mapping.xml]: All fixed
